1.8/instana-agent/README.md

```diff
@@ -0,0 +1,33 @@
+# Instana agent deployment on DC/OS
+
+[Instana](https://www.instana.com "Instana") is the next generation Application Performance Management solution that automatically monitors dynamic modern applications, including scheduled environments. Instana comprehensively manages your application’s quality of service with zero to absolute minimal configuration effort.
+
+The Instana agent package can be installed to DC/OS setups using the official package from the DC/OS universe.
+
+* **Estimated time for completion**: 2 minutes
+* **Scope**: Learn how to install the Instana agent on a DC/OS Cluster.
+
+## Prerequisites
+
+* A running DC/OS 1.8 cluster
+* Your Instana agent key (obtainable via the UMP menu in the Instana WebUI)
+
+## Deploying the Instana agent
+
+![Installing the Instana agent on DC/OS](img/install.png "Installing the Instana agent on DC/OS")
+
+In order to install the Instana agent, go to the Universe listing in the DC/OS web UI, and **click on "Install"** on the Instana agent list item. Since you need to provide custom informations, **click "Advanced Installation"**. The only required action here is to **fill in your Instana agent key**.
+
+![Provide your Instana agent key in the advanced tab](img/params1.png "Provide your Instana agent key in the advanced tab")
+
+If you're a SaaS customer, that is reporting to the US environment, the rest of the values can be left with the default values. EU - SaaS customers should **switch to the "saas-eu-west-1.Instana.io" endpoint**. If you're an on-prem customer, please **enter your backend's host address** and **set the port** (default: 1444). 
+
+![Switching to another reporting endpoint](img/params2.png "Switching to another reporting endpoint")
+
+You can **set host tags** in the tag_list (comma separated).
+
+## Scaling the Instana agent
+
+This is being taken care of by the deployment constraints. If you have, say, three hosts, and set the scaling to three, DC/OS will deploy exactly one Instana agent on each host. If you want different host tags for each and every machine, the Instana agent needs to be deployed separately.
+
+![The Instana agent supports scaling](img/tasklisting.png "The Instana agent supports scaling")
```

Comment:
Please add a section for additional resources at the end. Here is an example: https://github.com/dcos/examples/tree/master/1.8/neo4j#further-information

Apart from that, PR LGTM!